captura/views.py

```diff
@@ -1,3 +1,53 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from django.contrib.auth.decorators import user_passes_test, login_required
+from perfiles_usuario.utils import is_capturista
 
-# Create your views here.
+
+@login_required
+@user_passes_test(is_capturista)
+def pending_studies(request):
+    """ View to see the list of pending studies.
```

Comment:
I guess all of these are fine, but can you add a TODO annotation indicating that this is just a dummy view?